db/migrate/20150819150242_add_more_fields_to_project_details.rb

```diff
@@ -0,0 +1,133 @@
+class AddMoreFieldsToProjectDetails < ActiveRecord::Migration
+  def up
+    execute <<-SQL
+      drop function if exists public.published_states();
+      CREATE FUNCTION public.published_states() RETURNS text[]
+        LANGUAGE sql STABLE SECURITY DEFINER
+          AS $$
+            SELECT '{"online", "waiting_funds", "failed", "successful"}'::text[];
+          $$;
+
+      drop function if exists public.is_published(projects);
+      create function public.is_published(projects) returns boolean
```

Comment:
We should rewrite the model method:
``` 
published?
```
using this function to avoid duplicated logic between database and ruby